pepr/attack.py

```diff
@@ -38,6 +39,14 @@ def __init__(
         self.target_models = target_models
         self.attack_results = {}
 
+        # Dummy section to get rid of unresolved reference warnings. This attribute
+        # should be overwritten by the actual report section as soon as it is created.
```

Comment:
When do this reference warnings occur?

Reply:
Linter warnings in the wrappers everywhere we use `report_section`